### ISSUE TITLE
Use pre-trained model to extract feature vectors

### ISSUE THREAD

**Issue by liunancun** [deeplearning4j/deeplearning4j#9388]
import numpy as np
from tensorflow.keras.applications.resnet50 import ResNet50
from tensorflow.keras.applications.resnet50 import preprocess_input as preprocess_input_resnet50
from tensorflow.keras.preprocessing import image
from numpy import linalg as LA

img = image.load_img("D:\\demo.png", target_size=(224, 224))
img = image.img_to_array(img)
img = np.expand_dims(img, axis=0)
img = preprocess_input_resnet50(img)

model = ResNet50(weights='imagenet', input_shape=(224, 224, 3), pooling="max", include_top=False)

feat = model.predict(img)

norm_feat = feat[0] / LA.norm(feat[0])

print(norm_feat)

I have a piece of code written in Python to extract feature vectors. How can I convert it into Deeplearning4j?

**Comment by agibsonccc** [comment]
Closing as this is mainly a support issue. Please use our forums for questions like this. I would suggest looking at our keras import.  https://deeplearning4j.konduit.ai/keras-import/overview
